lib/Echidna/Campaign.hs

```diff
@@ -235,14 +236,30 @@ addToCorpus res = unless (null rtxs) $ hasLens . corpus %= (rtxs:) where
 randseq :: ( MonadCatch m, MonadRandom m, MonadReader x m, MonadState y m
            , Has GenDict y, Has TxConf x, Has TestConf x, Has CampaignConf x, Has Campaign y)
         => Int -> Map Addr Contract -> World -> m [Tx]
-randseq ql o w = do 
+randseq ql o w = do
   ca <- use hasLens
-  let txs = ca ^. corpus  
-      p   = ca ^. ncallseqs 
-  if length txs > p then -- Replay the transactions in the corpus, if we are executing the first iterations
-    return $ txs !! p
-  else                   -- Randomly generate new transactions 
-    replicateM ql (evalStateT (genTxM o) (w, ca ^. genDict))
+  let ctxs = ca ^. corpus
+      p    = ca ^. ncallseqs
+  if length ctxs > p then -- Replay the transactions in the corpus, if we are executing the first iterations
+    return $ ctxs !! p
```

Comment:
i think in the future we want to rethink this. we should preload the `GenDict` with values obtained from executing corpus transactions and populate the coverage dictionary accordingly.

Reply:
Yes, we could do that. But this simple approach guarantees that the corpus transactions are tested before anything else. In that case:
* The tests can fail before starting with the actual fuzzing campaign.
* The coverage will be as high as possible. This is useful to keep a "stable" corpus that will grow over time when new coverage is detected.

Reply:
yeah i'm not saying change it now. but a future PR could improve this